internal/testdata/objectives.go

```diff
@@ -70,7 +75,15 @@ func genericVFO() virtualfund.Objective {
 		ts.Outcome,
 		ts.ChannelNonce.Int64(),
 	}
-	testVFO, err := virtualfund.NewObjective(request, Channels.MockTwoPartyLedger, Channels.MockConsensusChannel)
+
+	ledgerPath := createLedgerNetwork([]testactors.Actor{
+		testactors.Alice,
+		testactors.Irene,
+		testactors.Bob,
+	}, nil)
```

Comment:
Is it useful to construct this entire network but then only return one participant's "view" on it? I would have thought the real utility of your helpers was to bootstrap the ledger network plus an objective for each participant? Otherwise, if I want Bob's objective then I will create the ledger network all over again?

Reply:
I think this code could be reused / supplemented for that purpose, yes. Maybe via something like:

```
func (a Actor) vfo(counterparty Actor, network LedgerNetwor) virtualfund.Objective { ... }
```

The goal of the moment is just to accurately construct a single vfo which can be written and retrieved from the store. I found it easier to think about instantiating ledger channels in context of other ledger channels rather than as a one-off, and thought it'd be a good time to write this code.